compiler/parser/RdrHsSyn.hs

```diff
@@ -3250,7 +3250,7 @@ mkInterfaceDecl tycon decls = do
                         , con_ex_tvs = []
                         , con_mb_cxt = Nothing
                         , con_args = PrefixCon [ghcTypesOpaque]
-                        , con_doc = Nothing
+                        , con_doc = mbConDoc
```

Comment:
I’m a bit surprised we need this. My understanding was that we get documentation by matching up doc comments to the nearest identifier. So if that doesn’t work for interfaces, maybe we’re losing the source location?

Reply:
no, I was just testing. We don't need it, it works out of the box,